.travis.yml

```diff
@@ -4,15 +4,13 @@ branches:
     - latest
 node_js:
   - '8'
-  - '10'
```

Comment:
You removed node.js version 10, is there a specific reason why you had to remove it?

Reply:
This package is very old and the integration are very fragile and runs in parallel but share the same databases which makes them fail intermittently(for instance one test add something which breaks other parallel test case's which do not expect anything). So after a lot of debugging I made it green by disabling 10 and some other things.

Reply:
That is fine. 👍🏾